### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wellloy1**
Fixed a mistake. "}" is accidentally missed. The example with that mistake will not work.

### CODE REVIEW

**Comment by wellloy1** [comment]
Fixed a mistake. "}" is accidentally missed. The example with that mistake will not work.

File: README.md [hunk 1]
```diff
@@ -539,14 +539,15 @@ container.loadModules([
...
   resolverOptions: {
-    // We can give these auto-loaded modules
-    // the deal of a lifetime! (see what I did there?)
-    // By default it's `TRANSIENT`.
-    lifetime: Lifetime.SINGLETON,
-    // We can tell Awilix what to register everything as,
-    // instead of guessing. If omitted, will inspect the
-    // module to determine what to register as.
-    register: awilix.asClass
+      // We can give these auto-loaded modules
```
Comment (jeffijoe) [inline README.md:542]: Noticed the indentation is off now; let's change line 552 instead from `)` to `})`

Comment (wellloy1) [reply README.md:542]: Okay. But it will be line 551, as I see now.

**Comment by jeffijoe** [comment]
Thank you @wellloy1 🙏

**Pull request merged**